Websocket/socket.ts

```
@@ -93,17 +91,67 @@ export function setupWebSocketServer(wss: WebSocketServer) {
             }
+            break;
 
-            // Broadcast to all clients
-            broadcast({
-              type: "console_input",
-              data: data.data,
-              timestamp: new Date().toISOString(),
-            });
+          case "get_pending_prompts":
+            // Allow frontend to request current pending prompts
+            const pendingPrompts = ConsoleInputUtils.getPendingPrompts();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)